README.md

```diff
@@ -37,11 +41,17 @@ We are continuously adding more functionality to go-mysql-server. We support a s
 
 ## Custom functions
 
-- `IS_BINARY(blob)`: returns whether a BLOB is a binary file or not
+- `IS_BINARY(blob)`: returns whether a BLOB is a binary file or not.
+- `SUBSTRING(str,pos)`, ` SUBSTRING(str,pos,len)`: return a substring from the provided string.
+- Date and Timestamp functions: `YEAR(date)`, `MONTH(date)`, `DAY(date)`, `HOUR(date)`, `MINUTE(date)`, `SECOND(date)`, `DAYOFYEAR(date)`.
+- `ARRAY_LENGTH(json)`: If the json representation is an array, this function returns its size.
 
 ## Example
 
-`go-mysql-server` has a sql engine and a server implementation, so to start a server you must instantiate the engine and give it your `sql.Database` implementation that will be in charge to handle all the logic about retrieving the data from your source :
+`go-mysql-server` has a sql engine and a server implementation,
+so to start a server you must instantiate the engine and give it your `sql.Database` implementation.
+It will be in charge to handle all the logic about retrieving the data from your source.
```

Comment:
s/to handle/of handling/

Reply:
s/about retrieving/for retrieving/

Reply:
or maybe "...to retrieve the data from your source"